tutorials/physics/interpolation/using_physics_interpolation.rst

```diff
@@ -0,0 +1,72 @@
+
+We have tried to make the system as easy to use as possible, and many existing games will work with few changes. That said there are some situations which require special treatment, and these will be described.
+
+Turn on the physics interpolation setting
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The first step is to turn on physics interpolation in :ref:`ProjectSettings.physics/common/physics_interpolation<class_ProjectSettings_property_physics/common/physics_interpolation>`. You can now run your game.
+
+It is likely that nothing looks hugely different, particularly if you are running physics at 60 tps or a similar high tick rate, however quite a bit more is happening behind the scenes.
+
+.. tip:: To convert an existing game to use interpolation, it is highly recommended that you temporarily set :ref:`ProjectSettings.physics/common/physics_fps<class_ProjectSettings_property_physics/common/physics_fps>` to a low value such as 10, which will make interpolation problems more obvious.
+
+Move (almost) all game logic from _process to _physics_process
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The most fundamental requirement for physics interpolation (which you may be doing already) is that you should be moving and performing game logic on your objects within ``_physics_process`` (which runs at a physics tick) rather than ``_process`` (which runs on a rendered frame). This means your scripts should typically be doing the bulk of their processing within ``_physics_process``, including responding to input and AI.
+
+Setting the transform of objects only within physics ticks allows the automatic interpolation to deal with transforms *between* physics ticks, and ensures the game will run the same whatever machine it is run on.
+
+.. note:: If you attempt to set the transform of interpolated objects *outside* the physics tick, the calculations for the interpolated position will be incorrect, and you will get jitter (although this may not be visible on your machine, it will likely occur for some players). For this reason setting the transform of interpolated objects should be avoided outside of the physics tick, and Godot will attempt to produce warnings in the Editor if this case is detected.
+
+.. tip:: This is only a "soft-rule". There are some occasions where you might want to teleport objects outside of the physics tick (for instance when starting a level, or respawning objects), but in general you should be applying transforms from the physics tick.
+
+
+Ensure that all indirect movement happens during physics ticks
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Consider that in Godot, Nodes can be moved not just directly in your own scripts, but also by automatic methods such as tweening, animation, and navigation. All these methods *must also* have their timing set to operate on the physics tick rather than each frame ("idle").
```

Comment:
By the way, this can be a problem for animations that change both a transform and a color at the same time. Physics interpolation only affects transforms, not other properties such as modulation or opacity.

See also https://github.com/godotengine/godot/issues/51318 which is what could happen if you use a low physics tick rate and an AnimationPlayer running on the physics tick.

Reply:
Yes absolutely there are a number of cases like this. Ideally almost everything in the engine would have the capability for interpolation. It is also particularly important for sound, to avoid harsh clicking as objects move around (I don't know whether this is dealt with already in the audio with some kind of interpolation).

Reply:
I've now changed this to make this a bit clearer  and add the caveat that you point out that they can also be used to animate properties that are *not* interpolated.